---
id: abc123
description: abc123 rule
---

## Test Cases

### Passed

#### Passed Example 1

```html
<html>
<p>hello world</p>
```

### Failed

#### Failed Example 1

```html
<img src="no-alt.jpg">
```

### Inapplicable

#### Inapplicable Example 1

```xml
<math aria-hidden="true"></math>
```