---
id: abc123
description: abc123 rule
---

## Test Cases

### Passed

#### Passed Example 1

```html
<html>
  <img src="/test-assets/alt.jpg" alt="">
</html>
```

### Failed

#### Failed Example 1

```html
<img src="/test-assets/no-alt.jpg" />
```

### Inapplicable

#### Inapplicable Example 1

```xml
<math aria-hidden="true"></math>
```
